config-templates/google/gcp-compute-persistent-disk-csi-driver/1.0.4/metadata.json

```diff
@@ -37,10 +37,6 @@
             "name": "supported-k8s-versions",
             "value": "1.18, 1.19, 1.20"
```

Comment:
Lets update k8s versions from 1.21 to 1.25 though its not being used in code logic but just for user's information purpose

Reply:
These versions is based on CSI driver support. I guess they supported only these while developing this template